assets/src/edit-story/app/media/utils/useProcessVideo.js

```diff
@@ -108,6 +139,17 @@ function useProcessVideo({
         }
       };
 
+      const onUploadProgress = ({ resource }) => {
+        const oldResourceWithId = { ...resource, id: oldResource.id };
```

Comment:
Maintain the existing attachment id for now, until replaced with the new one at the end.